src/main/scala/com/gu/salesforce/messageHandler/MessageHandlerLambda.scala

```diff
@@ -82,6 +82,7 @@ trait MessageHandler extends Logging {
   def handleRequest(inputStream: InputStream, outputStream: OutputStream, context: Context): Unit = {
 
     logger.info(s"Salesforce message handler lambda ${Config.stage} is starting up...")
+    logger.info(s"using config from ${Config.bucket}/${Config.key}")
```

Comment:
added this logging just to make sure my new code is running, if anyone is opposed to it I can remove it